src/coco/repository/util/repositoryUtil.ts

```diff
@@ -0,0 +1,86 @@
+import RepositoryType from "../model/RepositoryType";
+import Repository from "../model/Repository";
+import { execSync } from "child_process";
+import Logger from "../../../log/Logger";
+
+const GET_GIT_REPOSITORY_COMMAND = "git config --local --get remote.origin.url";
+const GIT_REPO_URL_SUFFIX = ".git";
+const DOMAIN_GITHUB = "github.com";
+const DOMAIN_GITLAB = "gitlab.com";
+const DOMAIN_BITBUCKET = "bitbucket.org";
+const CUSTOM_DOMAIN_REG_EXP_GITHUB = /github.(.+).com/; // TODO
```

Comment:
`.`s need to be escaped. Also, are these custom domains always `.com`?

Reply:
1) I am opening a PR for this 👍 
2) As far as I found out, yes. There is possibility that may be wrong, but this just for autofill so missing some cases is not that important

Reply:
1 is done in https://github.com/zeplin/vscode-extension/pull/14